Dockerfile

```diff
@@ -1,15 +1,12 @@
-FROM alpine:3.12.0@sha256:a15790640a6690aa1730c38cf0a440e2aa44aaca9b0e8931a9f2b0d7cc90fd65
+FROM alpine:3.14.3@sha256:5e604d3358ab7b6b734402ce2e19ddd822a354dc14843f34d36c603521dbb4f9
 
 USER root
 
-RUN ["apk", "--no-cache", "upgrade"]
-RUN ["apk", "--no-cache", "add", "tini"]
-RUN ["apk", "--no-cache", "--repository=http://dl-cdn.alpinelinux.org/alpine/edge/testing", "add", "nginx-naxsi", "nginx-naxsi-mod-http-naxsi", "nginx-naxsi-mod-http-xslt-filter", "nginx-naxsi-mod-http-geoip"]
+RUN ["apk", "--no-cache", "add", "openssl", "tini", "nginx-mod-http-naxsi=1.20.2-r0", "nginx-mod-http-xslt-filter=1.20.2-r0", "nginx-mod-http-geoip=1.20.2-r0"]
```

Comment:
Is there a reason this no longer installs `nginx-naxsi` package? Looks like its only available from edge but wondering why its no longer being installed?

Reply:
It is installed as a direct dependency of the nginx-nod-http-naxsi as well as the other nginx-mod packages to the correct version. But we can also specify this explicitly.

Reply:
So basically, `nginx-mod-http-naxsi` is what `nginx-naxsi` used to be - the package is no longer available at the correct Nginx version as the package maintainers have changed. It installs the `nginx` dependency first and then the `nginx-mod-http-naxsi` is the NAXSI module that is installed on top of it. This is similar to our [pay-nginx-proxy image that does the same thing](https://github.com/alphagov/pay-nginx-proxy/blob/master/Dockerfile).